packages/nimbus/src/components/combobox/combobox.mdx

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
docs are coming soon, the combobox is in a state where reviews would be helpful without them though.  I can either add them in this PR or a separate PR, whichever people prefer.